source/blog/2017-01-09-ember-2-11-released.md

```diff
@@ -0,0 +1,292 @@
+  deprecates the `Ember.K` utility function. See pull request [#14751](https://github.com/emberjs/ember.js/pull/14751)
+  for additional details.
+
+- [RFC #191](https://github.com/emberjs/rfcs/blob/master/text/0191-deprecate-component-lifecycle-hook-args.md)
+  deprecates the private arguments passed to the component lifecycle hooks
+  (`didInitAttrs`, `didReceiveAttrs` and `didUpdateAttrs`). Please note that
+  this only deprecates the usage of the arguments passed to this hook, not the
+  hooks themselves. See pull request [#14711](https://github.com/emberjs/ember.js/pull/14711)
+  for additional details.
+
+For more details on the upcoming changes in Ember.js 2.12, please review the
+[Ember.js 2.12.0-beta.1 release page](https://github.com/emberjs/ember.js/releases/tag/v2.12.0-beta.1).
+
+---
+
+## Ember Data
+
+Ember Data is the offical data persistence library for Ember.js applications.
+
+### Changes in Ember Data 2.11
+
+TBK
+
+For more details on the changes in Ember Data 2.11, please review the
+[Ember Data 2.11.0 release page](https://github.com/emberjs/data/releases/tag/v2.11.0).
+
+### Upcoming changes in Ember Data 2.12
+
+TBK
+
+#### Deprecations in Ember Data 2.12
+
+TBK
+
+For more details on the upcoming changes in Ember Data 2.11, please review the
+[Ember Data 2.12.0-beta.1 release page](https://github.com/emberjs/data/releases/tag/v2.12.0-beta.1).
+
+---
+
+## Ember CLI
+
+Ember CLI is the command line interface for managing and packaging Ember.js
+applications.
+
+### Upgrading Ember CLI
+
+You may upgrade Ember CLI separately from Ember.js and Ember Data! To upgrade
+your projects using `yarn` run `yarn upgrade ember-cli`. To upgrade your
+projects using `npm` run `npm install --save-dev ember-cli`. After running the
+upgrade command run `ember init` inside of the project directory to apply the
+blueprint changes. You can view those changes for [applications here](https://github.com/ember-cli/ember-new-output/compare/v2.10.0...v2.11.0)
+and [addons here](https://github.com/ember-cli/ember-addon-output/compare/v2.10.0...v2.11.0).
+
+### Changes in Ember CLI 2.11
+
+Ember CLI 2.11 no longer supports Node.js 0.12 per the
+[Ember Node.js LTS Support policy](http://emberjs.com/blog/2016/09/07/ember-node-lts-support.html).
+This also applies to a litany of subprojects in the Ember community. Please
+upgrade your Node.js version. We recommend adopting the most-recently-released
+Node.js LTS.
+
+Following the mitigation section in the recent [security incident
+report](/blog/2016/12/14/security-incident-aws-s3-key-exposure.html) we have
+begun the process of migrating repositories in the [ember-cli GitHub organization](https://github.com/ember-cli)
+to be published automatically via their automated build systems. Ember CLI
+itself has not yet been migrated and as such continues to follow our
+[manual release process](https://github.com/ember-cli/ember-cli/blob/master/RELEASE.md).
+
+#### Ember No Longer Supplied Via `bower`
```

Comment:
:+1: also, the current working makes it sounds like we are dropping support for installing ember.js with `bower`, which is not true.

Perhaps "Installing Ember.js via npm or yarn" if you want to be extra clear